signing_service/signing_service/signing_service.py

```diff
@@ -267,7 +281,26 @@ def _handle_connection(
                 middleman_message_response = self._prepare_error_response(ErrorCode.UnexpectedMessage, exception)
             # If received frame is correct, validate transaction.
             else:
+                self._update_last_signed_transaction_date(self.last_signed_transaction)
```

Comment:
There is no need to pass `self.last_signed_transaction` here, you can use it directly in this method.

Reply:
Done